src/installer.js

```diff
@@ -80,6 +80,13 @@ function getOptions (data, defaults) {
   // `description-line-too-long`.
   options.productDescription = wrap(options.productDescription, {width: 80, indent: ''})
 
+  // Create array with unique values from default & user-supplied dependencies
+  if (data.options) { // options passed programmatically
+    options.requires = _.union(defaults.requires, data.options.requires)
+  } else { // options passed via command-line
+    options.requires = _.union(defaults.requires, data.requires)
+  }
```

Comment:
Perhaps this should be a separate function so it can be tested via unit tests rather than an integration test.

The same thing should happen in `-debian` as well.

Reply:
Any suggestions for the name of the new function? `removeDuplicates`?